task/src/main/java/com/flipkart/flux/impl/task/LocalJvmTask.java

```diff
@@ -72,19 +69,28 @@ public int getExecutionTimeout() {
             While this works for methods with all unique param types, it
             will fail for methods where we have mutliple params of the same type.
              */
+            ClassLoader classLoader = toInvoke.getDeploymentUnitClassLoader() != null ? toInvoke.getDeploymentUnitClassLoader() : this.getClass().getClassLoader();
```

Comment:
When I wrote this code, I was under assumption that we would be using the same task module in isolation mode as well to execute and we need to have a default router which gives actors for execution. As we have discussed afterwards, task library won't be present in isolation mode, will change this code piece.